tests/src/core/testqgsadjacenttiles.cpp

```diff
@@ -0,0 +1,171 @@
+/***************************************************************************
+     testadjacenttiles.cpp
```

Comment:
Can I suggest renaming this test to testqgsvectorlayerrenderer, so that we can add other (non adjacent tiles) related tests in future?